### PR TITLE
feat(web): support sortOption for ReactiveList

### PR DESCRIPTION
**Pull request by mohdashraf010897**
**PR Type** 🏗️ `Enhancement`

**Description** Supports a `sortOption` key url-params to retain sort order for ReactiveList. https://github.com/appbaseio/reactivesearch/issues/261

[📔 Notion](https://www.notion.so/appbase/2a5bc1196ce6450793bd34f57ff6716b#625fdb9996d44a1caad3997ad345a524)

https://www.loom.com/share/ad2bcac00406439992ea9ccfc78fd7a0

### CODE REVIEW

File: packages/web/src/components/result/ReactiveList.js [hunk 1]
```diff
@@ -622,6 +614,18 @@ class ReactiveList extends Component {
 	};
 
 	updatePageURL = (page) => {
+		if (this.props.sortOptions) {
+			const sortOption = this.props.sortOptions[this.sortOptionIndex].label;
```
Comment (bietkul) [inline packages/web/src/components/result/ReactiveList.js]: Add a `null` check before reading `label` key.

**Review by bietkul — CHANGES_REQUESTED** [review]

**Comment by mohdashraf010897** [comment]
@bietkul 
https://www.loom.com/share/ae934d6e678a4b8184dd12fe152079a1

File: packages/web/src/components/result/ReactiveList.js (hunk 1 above)
Comment (mohdashraf010897) [reply packages/web/src/components/result/ReactiveList.js]: Fixed here:  https://github.com/appbaseio/reactivesearch/pull/1863/commits/2dce811ce4202c2bad7b37632f0843b118a78dad

**Comment by mohdashraf010897** [comment]
Demo for [feat(vue): support for sortOptions](https://github.com/appbaseio/reactivesearch/pull/1863/commits/50477e641be718168034a900d235262a7811a3fc)
https://www.loom.com/share/2a22581d6caa4d928e512c435015a246

**Review by bietkul — APPROVED** [review]

**Comment by bietkul** [comment]
@mohdashraf010897 The Vue demo that you shared doesn't reset the page on sort change.

**Comment by mohdashraf010897** [comment]
> @mohdashraf010897 The Vue demo that you shared doesn't reset the page on sort change.

Updated demo 
https://www.loom.com/share/71f665ef5de242d98ec536b36774d31c

also updated the basic example

**Review by bietkul — APPROVED** [review]

**Pull request merged**